Source/WebKit/Shared/API/c/WKContextMenuItemTypes.h

```diff
@@ -117,6 +117,9 @@ enum {
     kWKContextMenuItemTagMediaPlayPause,
     kWKContextMenuItemTagMediaMute,
     kWKContextMenuItemTagDictationAlternative,
+    kWKContextMenuItemPDFSinglePageContinuous,
+    kWKContextMenuItemPDFTwoPages,
+    kWKContextMenuItemPDFTwoPagesContinuous,
```

Comment:
New API/SPI values should be appended to the end (in this case, after `kWKContextMenuItemTagCopyCroppedImage`), in order to preserve binary compatibility.